content/shell/renderer/shell_content_renderer_client.cc

```diff
@@ -272,6 +271,51 @@ ShellContentRendererClient::CreateURLLoaderThrottleProvider(
   return std::make_unique<ShellContentRendererUrlLoaderThrottleProvider>();
 }
 
+#if BUILDFLAG(USE_STARBOARD_MEDIA)
+media::SupportedCodecs GetCobaltEmeSupportedCodecs() {
```

Comment:
Should we use the same naming rule, e.g., `GetStarboardEmeSupportedCodecs`?

Reply:
done